apollo-federation/src/query_graph/mod.rs

```diff
@@ -356,6 +399,11 @@ pub struct QueryGraph {
     /// lowered composition validation on a big composition (100+ subgraphs) from ~4 minutes to
     /// ~10 seconds.
     non_trivial_followup_edges: IndexMap<EdgeIndex, Vec<EdgeIndex>>,
+    // TODO: Are these duplicate fields??
+    // TODO: If not, write docs
+    subgraph_to_args: IndexMap<Arc<str>, Vec<ObjectFieldArgumentDefinitionPosition>>,
+    subgraph_to_arg_indices:
+        IndexMap<Arc<str>, IndexMap<ObjectFieldArgumentDefinitionPosition, String>>,
```

Comment:
I want to revisit this was we near the end of porting this feature. These fields contain the same data, in the same order. The only difference is one also contains identifier strings. We should be able to remove `subgraph_to_args`. For now, keeping it will make porting things easier.